src/components/Signup/Signup.js

```diff
@@ -77,6 +80,7 @@ const Signup = function ({
   location,
   dependencies: { dopplerLegacyClient, originResolver, localStorage, utmCookiesManager },
 }) {
+  useZohoScript({ scriptUrl });
```

Comment:
No entiendo la ventaja de esto sobre Helmet, o incluso, sobre to approach inical.

Reply:
No creo que haya una ventaja. Solo es una solución. 
Con helmet me encontré en el siguiente escenario: 
- **Se agrega el campo zc_gad** (campo oculto) al signup form. 
- **Se carga el script de seguimiento**: este script hace varias cosas, pero en general, guarda el parametro gclid en una cookie, luego hace una busqueda de todos los formularios que hay en el documento y por cada formulario que tenga el campo oculto de zoho llamado zc_gad le setea el valor a lo que tenga gclid (por eso es que se cerro el pull request del campo gclid, porque con este task se soluciona lo del gclid, ya que este valor se almacena en zc_gad).

El problema: el script de seguimiento, no estaba almacenando el valor del parametro gclid en zc_gad. La razón es que apesar de que el form estaba presente en el documento, el no lo conseguía. Es como si se estuviera cargando el script antes de que se cargara el html del campo oculto. Probé con una carga diferida (atributo defer al tag de script) y tampoco funcionó (se supone que esto ejecuta el script al finalizar el análisis del doumento).

Reply:
Tal vez se pueda probar con React.Lazy. Eso veíamos con Junior.

Reply:
Mmm, no se si lo que se quita de https://github.com/FromDoppler/doppler-webapp/pull/1390 es lo mismo que se agrega acá.

De hecho, lo que se hace acá parece ser mucho más grande, pero se usa realmente? me gustaría juntarme con ustedes y revisarlo.